packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -930,6 +930,13 @@ export interface DatabaseInstanceProps extends DatabaseInstanceSourceProps {
    * @default - default master key if storageEncrypted is true, no key otherwise
    */
   readonly storageEncryptionKey?: kms.IKey;
+
+  /**
+   * Indicates whether the DB instance is an internet-facing instance
+   *
+   * @default - no default, behavior depends on your VPC setup and the database subnet group.
```

Comment:
Should be moved to `DatabaseInstanceNewProps`.

and:
```suggestion
   * @default - `true` if the instance is deployed in a `PUBLIC` subnet, `false` otherwise.
```